subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/VariantTransform.java

```diff
@@ -18,24 +18,13 @@
 
 import org.gradle.api.Action;
 import org.gradle.api.ActionConfiguration;
-import org.gradle.api.attributes.AttributeContainer;
 
 /**
  * Registration of an variant transform.
  *
  * @since 3.5
  */
-public interface VariantTransform {
-    /**
-     * Attributes that match the variant that is consumed.
-     */
-    AttributeContainer getFrom();
-
-    /**
-     * Attributes that match the variant that is produced.
-     */
-    AttributeContainer getTo();
-
+public interface VariantTransform extends ArtifactTransformSpec {
```

Comment:
This inheritance looks surprising...

Reply:
I did change this comment before submitting, but it stayed like this for some reason. Weird.

Reply:
Anyway, what I wanted to say here is: since this type is going out of business, I think we shouldn't extend `ArtifactTransformSpec` but keep the `getTo()` and `getFrom()` methods defined here. Unless we actually use it as an `ArtifactTransformSpec` which I think we do not, right?

Reply:
I removed the inheritance and duplicated the methods in the old and new types.